otx/algorithms/detection/adapters/mmdet/utils/config_utils.py

```diff
@@ -346,6 +346,10 @@ def patch_tiling(config, hparams, dataset=None):
             logger.info(f"Patch model from: {config.model.type} to CustomMaskRCNNTileOptimized")
             config.model.type = "CustomMaskRCNNTileOptimized"
 
+            for subset in ("val", "test"):
+                if config.data[subset].pipeline[0]["transforms"][-1]["type"] == "Collect":
+                    config.data[subset].pipeline[0]["transforms"][-1]["keys"].append("full_res_image")
+
```

Comment:
Can we just add "full_res_image" to tile_pipeline.py directly?

Reply:
Then we will need to add it to every `simple_test` that supports tiling. I added this only when the tile classifier is enabled, so the change is only focused on this issue.